shard-operator-parent/shard-operator-v2/src/test/java/com/redhat/service/smartevents/shard/operator/v2/ManagedBridgeServiceImplTest.java

```diff
@@ -0,0 +1,175 @@
+package com.redhat.service.smartevents.shard.operator.v2;
+
+import java.time.Duration;
+
+import javax.inject.Inject;
+
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import com.redhat.service.smartevents.infra.v2.api.models.OperationType;
+import com.redhat.service.smartevents.infra.v2.api.models.dto.BridgeDTO;
+import com.redhat.service.smartevents.shard.operator.core.providers.GlobalConfigurationsConstants;
+import com.redhat.service.smartevents.shard.operator.core.resources.knative.KnativeBroker;
+import com.redhat.service.smartevents.shard.operator.core.utils.LabelsBuilder;
+import com.redhat.service.smartevents.shard.operator.v2.providers.NamespaceProvider;
+import com.redhat.service.smartevents.shard.operator.v2.resources.ManagedBridge;
+import com.redhat.service.smartevents.shard.operator.v2.utils.Fixtures;
+import com.redhat.service.smartevents.shard.operator.v2.utils.V2KubernetesResourcePatcher;
+
+import io.fabric8.kubernetes.api.model.ConfigMap;
+import io.fabric8.kubernetes.api.model.HasMetadata;
+import io.fabric8.kubernetes.api.model.Secret;
+import io.fabric8.kubernetes.client.KubernetesClient;
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.kubernetes.client.WithOpenShiftTestServer;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@QuarkusTest
+@WithOpenShiftTestServer
+public class ManagedBridgeServiceImplTest {
```

Comment:
Missing test for `deleteManagedBridge`, `fetchOrCreateBridgeSecret`, `fetchOrCreateBridgeConfigMap`, `fetchOrCreateKnativeBroker` methods

Reply:
@vaibhavjainwiz Delete will be covered by https://issues.redhat.com/browse/MGDOBR-1246

Reply:
The other methods are tested indirectly at the minute and I think we may end up refactoring some of it further, so I'll leave it for this PR.